Update AlphaAnimals_CE_Patch_Race_Aura.xml

Breadbox2k19 opened the pull request:
Differentiate Aura from Scyther by making it less armored, but faster, deal less damage per hit for more frequent attacks.
Various melee stats brought to par(or better) of an average scyther to make it more deadly in melee.

Breadbox2k19 committed:
Update AlphaAnimals_CE_Patch_Race_Aura.xml

Differentiate Aura from Scyther by making it less armored, but faster, deal less damage per hit for more frequent attacks.
Various melee stats brought to par(or better) of an average scyther to make it more deadly in melee.

Juanfrank commented on Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Aura.xml:
```diff
@@ -23,11 +23,13 @@
+						<MeleeDodgeChance>0.33</MeleeDodgeChance>
+						<MeleeCritChance>0.24</MeleeCritChance>
+						<MeleeParryChance>0.33</MeleeParryChance>
+						<MaxHitPoints>200</MaxHitPoints>
```
I think the Aura body already does a similar MaxHitPoints pull through the `<li Class="AlphaBehavioursAndEvents.CompProperties_ChangeDef">` unless I'm wrong and it just takes default values or the sum of the bodyDef values.

Juanfrank reviewed (commented):
Playtest wait.

Juanfrank committed:
Update AlphaAnimals_CE_Patch_Race_Aura.xml

Very slightly decreased move speed and attack speed while also increasing his head damage a little.
